io/io/src/RRawFileUnix.cxx

```diff
@@ -88,6 +92,21 @@ void ROOT::Internal::RRawFileUnix::OpenImpl()
    }
 }
 
+void ROOT::Internal::RRawFileUnix::ReadVImpl(RIOVec *ioVec, unsigned int nReq)
+{
+#ifdef HAVE_LIBURING
```

Comment:
Instead of using ``` HAVE_LIBURING ``` lets try to add it as `R__HAS_URING`:  https://github.com/root-project/root/blob/master/config/RConfigure.in#L64 and https://github.com/root-project/root/blob/fdc718926ac5bedc82829128a781db0bd67c0c00/cmake/modules/RootConfiguration.cmake#L479 (use tmva-cpu CMake option as an example)

Reply:
Thank you @oshadura, done in 548199a :)